File: check-tor.php
<?php
$file = 'http://check.torproject.org/';
$searchfor = 'Congratulations. This browser is configured to use Tor.';

$contents = file_get_contents($file);
$pattern = preg_quote($searchfor, '/');
$pattern = "/^.*$pattern.*\$/m";

if(preg_match_all($pattern, $contents, $matches)){
   echo "\nTor is Configured!!\n";
}
else{
   echo "\nTor Not Configured!!\n";
}
?>

File: check-tor.py
import requests
import os
import sys
import re

word = "Congratulations. This browser is configured to use Tor." #THE SENTENCE SHOULD BE PRECISE
check = requests.get('https://check.torproject.org')
if word in check.text:
  print ("Tor Configured Successfully!")
else:
       print ("Connection not appears to be Routed via Tor!")
       exit()
